server/routerlicious/packages/services-shared/src/configDumper.ts

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import fastRedact from "fast-redact";
+import { ILogger } from "@fluidframework/server-services-core";
+import { Lumberjack } from "@fluidframework/server-services-telemetry";
+const errorSanitizationMessage = "FluidREDACTED";
+
+export class ConfigDumper {
+	private readonly config: Record<string, any>;
+	private readonly secretsList = [
+		"redis.pass",
```

Comment:
Are MongoDB endpoints/connection strings something we also want to add in the default secrets list?

https://github.com/microsoft/FluidFramework/blob/534a47f9cebea7c3fe6fa56cd880bbb25356a0f5/server/routerlicious/kubernetes/routerlicious/templates/fluid-configmap.yaml#L28-L29

Reply:
Thanks! I missed this.